h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -4283,3 +4283,78 @@ def saveModelMojo(model):
     os.makedirs(tmpdir)
     model.download_mojo(path=tmpdir)    # save mojo
     return tmpdir
+
```

Comment:
don't know if it's a good idea to put those functions in bloated `utilsPy`.
This file is already full of very specific functions that are used by only a few tests.
Wouldn't it be nicer to have a utility module in the `testdir_algos/glm` module?

Reply:
Yeah, it is good to refractor it.

Reply:
I removed the function that I used to glmUtils.py.  I did not put it in testdir_algos/glm because the run.py may not like it.